ngboost/ngboost.py

```diff
@@ -395,9 +395,24 @@ def feature_importances_(self):
             return None
         # Reshape the base_models
         params_trees = zip(*self.base_models)
+
         # Get the feature_importances_ for all the params and all the trees
+
+        def _get_feature_importance(tree, tree_index):
```

Comment:
re: your question [here](https://github.com/stanfordmlgroup/ngboost/issues/181#issuecomment-690458734): I don't see a reason why it needs to be an inner function so let's make this a method of the NGBoost class instead so that nobody gets confused later on.

Reply:
sounds good to me